qunit tests fail

From a fresh checkout, `npm install; npm test` gives the following error:

```
Running "qunit" task

Running "qunit:files" (qunit) task
Verifying property qunit.files exists in config...OK

Testing geolib.test.html

events.js:72
        throw er; // Unhandled 'error' event
              ^
Error: spawn ENOENT
    at errnoException (child_process.js:980:11)
    at Process.ChildProcess._handle.onexit (child_process.js:771:34)
npm ERR! weird error 8
npm ERR! not ok code 0
```

--- a/package.json
+++ b/package.json
@@ -15,7 +15,8 @@
     "devDependencies": {
         "grunt": "~0.3.17",
         "grunt-contrib-clean": "~0.3.0",
-        "grunt-contrib-copy": "~0.3.0"
+        "grunt-contrib-copy": "~0.3.0",
+        "phantomjs": "~1.9.0"
     },
     "licenses": [
         {
@@ -32,9 +33,9 @@
         "geo",
         "distance"
     ],
-	"scripts": {
-		"test": "grunt travis --verbose"
-	},
+    "scripts": {
+        "test": "grunt travis --verbose"
+    },
     "version": "1.2.8",
     "main": "./geolib"
 }

